frontend/packages/dev-console/integration-tests/support/constants/global.ts

```diff
@@ -53,18 +53,21 @@ export enum operators {
   AMQStreams = 'AMQ Streams',
   RHOAS = 'RHOAS',
   Jaeger = 'Red Hat OpenShift distributed tracing platform',
+  BuildsForOpenshiftOperator = 'builds for Red Hat OpenShift Operator',
 }
 
 export enum operatorNamespaces {
   PipelinesOperator = 'openshift-operators',
   ServerlessOperator = 'openshift-serverless',
   ShipwrightOperator = 'openshift-operators',
+  BuildsForOpenshiftOperator = 'openshift-operators',
```

Comment:
Can the ShipwrightOperator be used instead of BuildsForOpenshiftOperator as they have the same value

Reply:
Lets have both of them for this release. The reason is the Shipwright (community) is being currently moved from v1alpha1 to v1beta1. But I am not sure whether this is complete and I have not specifically tested. 
On the other hand, the Builds for Openshift is already launched as v1beta1.
Since both of them are not totally same as of now, lets keep both. 
Later on, once we confirm both works for all cases, we can remove the community operator.